### ISSUE TITLE
installing on openmediavault

### ISSUE THREAD

**Issue by raiden076** [hocus-dev/hocus#87]
## 🐛 Bug description

I am trying to install hocus on a old x86 machine, which is currently running openmediavault, 
this is the error I am getting, have no idea why:

------
> [setup-vm-images 3/6] RUN apt-get update && apt-get install -y curl wget qemu-system psmisc expect:
#7 156.7 E: Failed to fetch http://archive.ubuntu.com/ubuntu/pool/main/q/qemu/qemu-block-extra_6.2%2bdfsg-2ubuntu6.8_amd64.deb  Could not resolve 'archive.ubuntu.com' [IP: 91.189.91.38 80]
#7 156.7 E: Failed to fetch http://archive.ubuntu.com/ubuntu/pool/main/q/qemu/qemu-system_6.2%2bdfsg-2ubuntu6.8_amd64.deb  Could not resolve 'archive.ubuntu.com' [IP: 91.189.91.38 80]
#7 156.7 E: Failed to fetch http://archive.ubuntu.com/ubuntu/pool/main/libg/libglvnd/libglvnd0_1.4.0-1_amd64.deb  Could not resolve 'archive.ubuntu.com' [IP: 91.189.91.38 80]
#7 156.7 E: Failed to fetch http://archive.ubuntu.com/ubuntu/pool/main/libg/libglvnd/libglx0_1.4.0-1_amd64.deb  Could not resolve 'archive.ubuntu.com' [IP: 91.189.91.38 80]
#7 156.7 E: Failed to fetch http://archive.ubuntu.com/ubuntu/pool/main/libg/libglvnd/libgl1_1.4.0-1_amd64.deb  Could not resolve 'archive.ubuntu.com' [IP: 91.189.91.38 80]
#7 156.7 E: Failed to fetch http://archive.ubuntu.com/ubuntu/pool/main/q/qemu/qemu-system-gui_6.2%2bdfsg-2ubuntu6.8_amd64.deb  Could not resolve 'archive.ubuntu.com' [IP: 91.189.91.38 80]
#7 156.7 E: Failed to fetch http://archive.ubuntu.com/ubuntu/pool/main/e/edk2/ovmf_2022.02-3ubuntu0.22.04.1_all.deb  Could not resolve 'archive.ubuntu.com' [IP: 91.189.91.38 80]
#7 156.7 E: Failed to fetch http://archive.ubuntu.com/ubuntu/pool/main/e/edk2/qemu-efi-aarch64_2022.02-3ubuntu0.22.04.1_all.deb  Could not resolve 'archive.ubuntu.com' [IP: 91.189.91.38 80]
#7 156.7 E: Failed to fetch http://archive.ubuntu.com/ubuntu/pool/main/e/edk2/qemu-efi-arm_2022.02-3ubuntu0.22.04.1_all.deb  Could not resolve 'archive.ubuntu.com' [IP: 91.189.91.38 80]
#7 156.7 E: Unable to fetch some archives, maybe run apt-get update or try with --fix-missing?
------

## Environment

- Hocus version (commit hash) : installed through script today
- OS and its version: openmediavault 6

**Comment by gorbak25** [comment]
@raiden076 looks like a transient network failure with `archive.ubuntu.com`. Could you rerun the installation?

**Comment by raiden076** [comment]
@gorbak25 yes, I am retrying again, hocus is exactly what I wanted in my pipeline, if only it gets installed,
Is there any option to get verbose output, so troubleshooting further becomes a bit easier?

It's stuck in the vm builder step for the last 10 minutes.
---
dockeruser@omv:~/hocus$ HOCUS_HOSTNAME="omv" ops/bin/local-up.sh
ops/bin/local-up.sh: line 102: file: command not found
Building docker images 👷📦
Building vm-builder step #7 elapsed 23.42 s
---

**Comment by raiden076** [comment]
Tried as root too, gets stuck without any prompt at the same place:

```
root@omv:~/hocus# HOCUS_HOSTNAME="omv" ops/bin/local-up.sh
ops/bin/local-up.sh: line 102: file: command not found
Building docker images 👷📦
Building vm-builder step #7 elapsed 10.56 s
```
